### DISCUSSION TITLE
Projects based on your library

### DISCUSSION THREAD

**Discussion by awekrx** [acheong08/ChatGPT#988] [Show and tell]
Hello, I have 2 projects based on my library, I would like more people to know about them. Can you add them to your list?
https://github.com/awekrx/ChatGPT-MidJourney-prompt
https://github.com/awekrx/AutoDoc-ChatGPT

**Comment by acheong08** [comment]
This is exactly what I always wanted but never had the time to make! Thank you so much. 

> awekrx/AutoDoc-ChatGPT

I'll be using this to document this library
